client/src/data_layer_configs/lead_service_lines_by_water_systems_config.ts

```diff
@@ -39,6 +40,15 @@ const LEGEND_VALUES = [
   },
 ];
 
+const createLegends = (): Map<GeographicLevel, LegendInfo> => {
+  const waterSystemLegendInfo = {
+    title: 'Percent of service lines estimated to be lead',
+    buckets: LEGEND_VALUES,
+    bucketLabelType: FeaturePropertyDataType.Percentage,
+  };
+  return new Map([[GeographicLevel.State, waterSystemLegendInfo]]);
```

Comment:
should this still be state?

Reply:
The legends are the same for all levels of zoom since this is conveniently percentage. I added some comments explaining what's happening